libs/deer/desert/src/assert.rs

```diff
@@ -1,3 +1,5 @@
+#[cfg(feature = "pretty")]
+use alloc::format;
```

Comment:
This does not seem like an error the client is supposed to solve but the upstream library. What do you think?

Reply:
I want to fix the CI as soon as possible, but I will create a PR upstream soon to fix it instead. This will be a tiny bit more complicated, however.